core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/Sample.java

```diff
@@ -62,6 +65,8 @@
   public static final Long UNSAVED_ID = 0L;
   /** Field PREFIX */
   public static final String PREFIX = "SAM";
+  List<String> CATEGORIES = Collections.unmodifiableList(Arrays.asList(SampleIdentity.CATEGORY_NAME, SampleTissue.CATEGORY_NAME,
```

Comment:
Is there a reason to have this in plain `Sample` instead of `SampleClass` or `DetailedSample`?